src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -2032,11 +2032,12 @@ subroutine update_shelf_mass(G, US, CS, ISS, Time)
 end subroutine update_shelf_mass
 
 !> Save the ice shelf restart file
-subroutine ice_shelf_query(CS, G, frac_shelf_h)
+subroutine ice_shelf_query(CS, G, frac_shelf_h, mass_shelf)
   type(ice_shelf_CS),         pointer    :: CS !< ice shelf control structure
   type(ocean_grid_type), intent(in)      :: G  !< A pointer to an ocean grid control structure.
-  real, optional, dimension(SZI_(G),SZJ_(G))  :: frac_shelf_h !<
-                                      !< Ice shelf area fraction [nodim].
+  real, optional, dimension(SZI_(G),SZJ_(G))  :: frac_shelf_h !< Ice shelf area fraction [nodim].
```

Comment:
Please add the appropriate `intent()` qualifiers for the frac_shelf_h and mass_shelf arguments to `ice_shelf_query()`.  Also, the standard syntax for these two units would be `[nondim]` and `[R Z ~> kg m-2]`.